methods.ts

```diff
@@ -1849,19 +1872,6 @@ export interface InputSticker<F> {
   keywords?: string[];
 }
 
-/** This object represents the content of a media message to be sent. It should be one of
-  - InputMediaAnimation
-  - InputMediaDocument
-  - InputMediaAudio
-  - InputMediaPhoto
-  - InputMediaVideo */
-export type InputMedia<F> =
-  | InputMediaAnimation<F>
-  | InputMediaDocument<F>
-  | InputMediaAudio<F>
-  | InputMediaPhoto<F>
-  | InputMediaVideo<F>;
-
```

Comment:
I can see that these are back in the next commit but if you want to have a clean commit history, better to merge `Add methods and types for API 7.5` and `Adds PaidMedia types` commits. And also, move the `Paid Media` types to the `Add support for 7.6` commit. These are just suggestions.